greenheart/tools/eco/utilities.py

```diff
@@ -1444,6 +1442,1057 @@ def save_energy_flows(
+                                            'steel electrolysis Scope3 EI (kg CO2e/MT steel)': steel_electrolysis_Scope3_emission_intensity, 
+                                            'steel electrolysis Scope2 EI (kg CO2e/MT steel)': steel_electrolysis_Scope2_emission_intensity, 
+                                            'steel electrolysis Scope1 EI (kg CO2e/MT steel)': steel_electrolysis_Scope1_emission_intensity, 
+                                            'steel electrolysis EI (kg CO2e/MT steel)': steel_electrolysis_emission_intensity,
+                                            'steel smr Scope3 EI (kg CO2e/MT steel)': steel_smr_Scope3_emission_intensity, 
+                                            'steel smr Scope2 EI (kg CO2e/MT steel)': steel_smr_Scope2_emission_intensity, 
+                                            'steel smr Scope1 EI (kg CO2e/MT steel)': steel_smr_Scope1_emission_intensity,
+                                            'steel smr EI (kg CO2e/MT steel)': steel_smr_emission_intensity,
+                                            'steel smr ccs Scope3 EI (kg CO2e/MT steel)': steel_smr_ccs_Scope3_emission_intensity, 
+                                            'steel smr ccs Scope2 EI (kg CO2e/MT steel)': steel_smr_ccs_Scope2_emission_intensity, 
+                                            'steel smr ccs Scope1 EI (kg CO2e/MT steel)': steel_smr_ccs_Scope1_emission_intensity,
+                                            'steel smr ccs EI (kg CO2e/MT steel)': steel_smr_ccs_emission_intensity,
+                                            'steel atr Scope3 EI (kg CO2e/MT steel)': steel_atr_Scope3_emission_intensity, 
+                                            'steel atr Scope2 EI (kg CO2e/MT steel)': steel_atr_Scope2_emission_intensity, 
+                                            'steel atr Scope1 EI (kg CO2e/MT steel)': steel_atr_Scope1_emission_intensity,
+                                            'steel atr EI (kg CO2e/MT steel)': steel_atr_emission_intensity,
+                                            'steel atr ccs Scope3 EI (kg CO2e/MT steel)': steel_atr_ccs_Scope3_emission_intensity, 
+                                            'steel atr ccs Scope2 EI (kg CO2e/MT steel)': steel_atr_ccs_Scope2_emission_intensity, 
+                                            'steel atr ccs Scope1 EI (kg CO2e/MT steel)': steel_atr_ccs_Scope1_emission_intensity,
+                                            'steel atr ccs EI (kg CO2e/MT steel)': steel_atr_ccs_emission_intensity,
+                                            })
+    ## Interpolation of emission intensities for years not captured by cambium (cambium 2023 offers 2025-2050 in 5 year increments)
+    # Define end of life based on cambium_year and project lifetime
+    endoflife_year = cambium_year + project_lifetime
+
+    # Instantiate lists to hold interpolated data
```

Comment:
Perhaps similarly (could reuse the previous `technologies` and `intensity_types`)
```
# Define technologies, processes, and intensity types
technologies = ["electrolysis", "smr", "smr_ccs", "atr", "atr_ccs", "NH3_electrolysis", "NH3_smr", "NH3_smr_ccs", "NH3_atr", "NH3_atr_ccs", "steel_electrolysis", "steel_smr", "steel_smr_ccs", "steel_atr", "steel_atr_ccs"]
intensity_types = ["Scope3", "Scope2", "Scope1", "total"]

# Initialize lists and corresponding values
interpolated_data = {
    f"{tech}_{intensity}_interpolated": []
    for tech in technologies
    for intensity in intensity_types
}
```